ros/src/msgs/autoware_config_msgs/msg/ConfigNDTSlam.msg

```diff
@@ -0,0 +1,13 @@
+Header header
```

Comment:
I highly recommend switching to `dynamic_reconfigure`, instead of growing the dependency of these type of messages.

Reply:
I also think that it is better to use dynamic_reconfigure.
However, at present, there are few nodes that use dynamic_reconfigure in Autoware.
If we mix both Autoware Runtime Manager's config and dynamic_reconfigure, I think users will cause confusion.
So I think I will not switch now.